src/iOS.Autofill/LockPasswordViewController.cs

```diff
@@ -9,6 +9,7 @@ public LockPasswordViewController(IntPtr handle)
             : base(handle)
         {
             BiometricIntegrityKey = "autofillBiometricState";
+            DismissModalAction = () => Cancel();
```

Comment:
You can assign the `Cancel` method directly for this delegate vs. wrapping it in an anonymous function.
i.e.
```c#
DismissModalAction = Cancel;
```